src/Xamarin.Android.Build.Tasks/Tasks/GenerateJavaStubs.cs

```diff
@@ -473,23 +484,21 @@ bool CreateJavaSources (IEnumerable<JavaType> newJavaTypes, TypeDefinitionCache
 
 				using (var writer = MemoryStreamPool.Shared.CreateStreamWriter ()) {
 					try {
-						var jti = new JavaCallableWrapperGenerator (t, Log.LogWarning, cache, classifier) {
```

Comment:
Since we don't pass in `Log.LogWarning()`, I wondered if we lost the ability to log warnings. But then I looked at the java.interop diff to see all the warnings commented out.

Is this something being worked on for a future PR? Just curious.

Reply:
Not that I know of.  I assume these warnings were disabled for a reason?